src/main/java/frc/team2412/robot/util/MatchDashboard.java

```diff
@@ -20,7 +20,6 @@ public MatchDashboard(Subsystems s) {
 		tab.add(new FMSWidget()).withPosition(0, 0).withSize(4, 1);
 		tab.add(field).withPosition(0, 1).withSize(4, 3);
 		Robot r = Robot.getInstance();
-		tab.add("Auto Chooser", r.autoChooser).withPosition(4, 0).withSize(2, 1);
-		AutonomousField.configureShuffleboardTab(tab, 6, 0, "Auto Chooser", r::addPeriodic);
+		AutonomousField.configureShuffleboardTab(tab, 6, 0, "Available Auto Variants", r::addPeriodic);
```

Comment:
I accidentally hardcoded `configureShuffleboardTab` to use "Auto Chooser" as the name instead of the `autoChooserName` argument [when making the subscriber](https://github.com/robototes/Crescendo2024/blob/main/src/main/java/frc/team2412/robot/util/AutonomousField.java#L39), so that will need to be fixed.